database connection

```diff
@@ -0,0 +1,28 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+from torchvision import datasets, transforms
+from torch.autograd import Variable
+
+# download and transform train dataset
+train_loader = torch.utils.data.DataLoader(datasets.MNIST('../mnist_data', 
+                                                          download=True, 
+                                                          train=True,
+                                                          transform=transforms.Compose([
+                                                              transforms.ToTensor(), # first, convert image to PyTorch tensor
+                                                              transforms.Normalize((0.1307,), (0.3081,)) # normalize inputs
+                                                          ])), 
+                                           batch_size=10, 
+                                           shuffle=True)
+
+# download and transform test dataset
+test_loader = torch.utils.data.DataLoader(datasets.MNIST('../mnist_data', 
+                                                          download=True, 
+                                                          train=False,
+                                                          transform=transforms.Compose([
+                                                              transforms.ToTensor(), # first, convert image to PyTorch tensor
+                                                              transforms.Normalize((0.1307,), (0.3081,)) # normalize inputs
+                                                          ])), 
+                                           batch_size=10, 
+                                           shuffle=True)
```

Comment:
Do we need the rest of the code from pytorch mnist repository for when we adapt to the python base?